specification/guestconfiguration/resource-manager/Microsoft.GuestConfiguration/stable/2020-06-25/guestconfiguration.json

```diff
@@ -0,0 +1,1317 @@
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true,
+          "description": "End date and time of the guest configuration assignment compliance status check."
+        },
+        "complianceStatus": {
+          "type": "string",
+          "readOnly": true,
+          "description": "A value indicating compliance status of the machine for the assigned guest configuration.",
+          "x-ms-enum": {
+            "name": "complianceStatus",
+            "modelAsString": true
+          },
+          "enum": [
+            "Compliant",
+            "NonCompliant",
+            "Pending"
+          ]
+        },
+        "operationType": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Type of report, Consistency or Initial",
+          "x-ms-enum": {
+            "name": "type",
+            "modelAsString": true
+          },
+          "enum": [
+            "Consistency",
+            "Initial"
+          ]
+        },
+        "resources": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AssignmentReportResourceMetadata"
+          },
+          "description": "The list of resources for which guest configuration assignment compliance is checked."
+        }
+      }
+    },
+    "GuestConfigurationAssignmentProperties": {
+      "properties": {
+        "targetResourceId": {
+          "type": "string",
+          "readOnly": true,
+          "description": "VM resource Id.",
+          "x-nullable": true
+        },
+        "subscriptionId": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Subscription Id to which the Guest Configuration assignment resource is assigned to.",
+          "x-nullable": true
+        },
+        "guestConfiguration": {
+          "$ref": "#/definitions/GuestConfigurationNavigation",
+          "description": "The guest configuration to assign."
+        },
+        "complianceStatus": {
+          "type": "string",
+          "readOnly": true,
+          "description": "A value indicating compliance status of the machine for the assigned guest configuration.",
+          "x-ms-enum": {
+            "name": "complianceStatus",
+            "modelAsString": true
+          },
+          "enum": [
+            "Compliant",
+            "NonCompliant",
+            "Pending"
+          ]
+        },
+        "lastComplianceStatusChecked": {
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true,
+          "description": "Date and time when last compliance status was checked.",
+          "x-nullable": true
+        },
+        "latestReportId": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Id of the latest report for the guest configuration assignment. ",
+          "x-nullable": true
+        },
+        "latestAssignmentReport": {
+          "$ref": "#/definitions/AssignmentReport",
```

Comment:
>AssignmentReport [](start = 33, length = 16)

why are there different definitions for assignmentReport?  Whats the difference between them?

Reply:
removed it - that wasn't needed